### ISSUE TITLE
happy to help get mongodb_ecto to work more easily.

### ISSUE THREAD

**Issue by cortfritz** [zookzook/elixir-mongodb-driver#248]
I can't get this to work and am probably missing something obvious.

I haven't gotten beyond this issue: mongodb_ecto seems to expect Mongo.Query struct to have the key `:extra`.  

If I could get this working I would love to submit PRs for explicit instructions in README or additional md file, and submit an example in examples/mongo_ecto.ex.

If :mongodb_driver needs to be updated to add this key to the query I'm also happy to submit that PR, and any insight into what `:extra` is doing would be appreciated.

**Comment by zookzook** [comment]
I think there is no reason to change anything here. You need to check this PR: https://github.com/elixir-mongo/mongodb_ecto/pull/190

**Comment by cortfritz** [comment]
Thank you zz.

Was `extra: coll` and subsequent references to `coll` vestigal or unused?  The PR removes it from the Query struct and comments out lines where it's referenced.  Maybe `coll` is already assumed in the context of a query.  Maybe `extra` allowed for an override of that?  I'm trying to figure out what might now be incompatible with this change.

**Comment by brennana** [comment]
@cortfritz Yes, the `extra: coll` was a remnant from the deprecated-and-removed mongo query commands. The new command structure, required by 6.0+, includes collections in the command log lines via the subcommand portion of the command. For example, the new commands now look like:

```
COMMAND [find: "my_collection", filter: [_id: #BSON.ObjectId<5cabe2398fbb653f09ffaf82>], projection: %{_id: true, created_at: true}, "$readPreference": %{mode: :primary}] []
```

where `find: "my_collection"` identifies the collection under the find subcommand. Aggregations and other collection commands work the same way.

The mongodb_ecto driver, as of the aforementioned PR#190, no longer expects the `extra` field in order to support MongoDB 6.0+, and the newer commands.

**Comment by scottmessinger** [comment]
@cortfritz We just merged in the change so the Elixir Ecto driver (rc.2) now works with this driver. We've had it in production and haven't had any issues.

**Comment by cortfritz** [comment]
> [@cortfritz](https://github.com/cortfritz) We just merged in the change so the Elixir Ecto driver (rc.2) now works with this driver. We've had it in production and haven't had any issues.

sorry for not replying!  just seeing this now.

thank you!
